syntax = "proto3";

enum GroupType {
  object_group = 0;
  process_group = 1;
}

message GROUP {
  string group_name = 1;
  repeated int32 entity_id = 2 [packed = true];
  string group_type = 3;
  int64 tmo_id = 4;
}